components/src/Icon/Icon.js

```diff
@@ -33,10 +33,11 @@ Svg.propTypes = {
   name: PropTypes.oneOf(iconNames).isRequired,
   size: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
   title: PropTypes.string,
-  color: PropTypes.string.isRequired,
+  color: PropTypes.string,
 };
 
 Svg.defaultProps = {
+  color: "currentColor",
```

Comment:
I took a look at the `InlineIcon` proptypes, and it looks like they're not needed anymore.
Could you remove those?